tests/static-analysis/Type/intersection.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+use Psl\Collection\Map;
+use Psl\Result\ResultInterface;
+use Psl\Type;
+
+/**
+ * @psalm-suppress UnusedParam
```

Comment:
p.s: you can rename the argument to `$_value` and psalm won't annoy you about unused argument ;)